server/db/create_test_data.js

```diff
@@ -115,6 +116,9 @@ program
   .parse(process.argv);
 
 var count = program.count || defaultCount;
-truncateTables(function() {
+truncateTables(function(err) {
+  if (err) {
+    throw err;
+  }
```

Comment:
If we get an error when trying to truncate tables, don't even bother trying to create users in the sketchy DB.